.github/workflows/pr-feedback.yml

```diff
@@ -26,4 +26,4 @@ jobs:
             Thank you for contributing to Nextcloud and we hope to hear from you soon!
           days-before-feedback: 14
           start-date: "2023-05-01"
-          exempt-authors: "${{ steps.scrape.outputs.users }},backportbot-nextcloud,dependabot,renovate"
+          exempt-authors: "${{ steps.scrape.outputs.users }},app/backportbot-nextcloud,app/dependabot,app/renovate"
```

Comment:
Not sure if it works, but could we exclude `app/*`?

Reply:
Not like this. Would have to be in the action code, then, would probably make sense